goentri.com.hotmart-subdomains.json

```diff
@@ -0,0 +1,27 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "hotmart-subdomains",
+  "serviceName": "Hotmart Subdomains",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "primaryIp is the IP, cnamehost is the host for the CNAME and cnamePointsTo is the URL to where the subdomain is pointed ",
```

Comment:
the variable description does not match the variables in the template

Reply:
@pawel-kow sorry for that, I updated the variables on that template.